backend/src/test/resources/org/cryptomator/hub/flyway/V9999__Test_Data.sql

```diff
@@ -55,3 +55,17 @@ VALUES
 	('device1', '7E57C0DE-0000-4000-8000-000100001111', 'jwe1'),
 	('device2', '7E57C0DE-0000-4000-8000-000100001111', 'jwe2'),
 	('device3', '7E57C0DE-0000-4000-8000-000100002222', 'jwe3'); -- device3 of user1, part of group1, has access to vault2
+
+INSERT INTO "audit_event" ("id", "timestamp", "type")
+VALUES
+    (999, '1970-01-01T00:00:02.900Z', 'UNLOCK'),
+    (1000, '1970-01-01T00:00:03Z', 'UNLOCK'),
+    (1111, '1970-01-01T00:00:04Z', 'UNLOCK'),
+    (4242, '1970-01-01T00:00:05Z', 'UNLOCK');
```

Comment:
since we try to use some "speaking" IDs in the test data, let's try to find some better values.

Reply:
@overheadhunter Do you have any suggestios? Due to the datatype we have to use a number. We can use hex notation to "write" something, but apart from that, i can't imagine to make the ids more "speakable".

The reason for picking such high numbers is, that other integrations-tests might fail using "unlock" might fail, because our sequence generator begins at 1 (or zero), but the table contains already entries for those low ids.

Reply:
Should not keep us from merging. Test data can be adjusted at any time, since this is no "real" migration.